conjurapi/router_v5.go

```diff
@@ -131,18 +131,37 @@ func (r RouterV5) LoadPolicyRequest(mode PolicyMode, policyID string, policy io.
 	)
 }
 
-func (r RouterV5) RetrieveBatchSecretsRequest(variableIDs []string) (*http.Request, error) {
+func (r RouterV5) RetrieveBatchSecretsRequest(variableIDs []string, base64Flag ...bool) (*http.Request, error) {
+	var base64Encode bool
+	if len(base64Flag) > 0 {
+		base64Encode = base64Flag[0]
+	}else{
+		base64Encode = false
+	}
```

Comment:
Go style for "ternary with if" is:

```go
base64Encode := false
if len(base64Flag) > 0 {
    base64Encode = base64Flag[0]
}
```
I also agree with @izgeri 's hesitation about using variadic args like this.  There is a good place for variadic args in the [functional options pattern](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis), but in this case, is there a good reason not to just force the caller to be explicit and always specify the flag?

Reply:
I changed the API so there are two different functions, one which uses the base64 encoding header and one which does not. I think exposing a flag to the user might be a more representative solution, however I didn't want to make any breaking changes in the API and this seems to be the best solution keeping that in mind.

Reply:
> I think exposing a flag to the user might be a more representative solution

Reply:
To me adding the header to the request is less like new functionality and more like an additional flag or option you can specify for the endpoint. However I just renamed the new method to `RetrieveBatchSecretsSafe` and I think it is a good alternative. Sharing the same root name indicates that both methods use the same endpoint, but the `Safe` keyword tells the user there is some additional logic in the request.

Reply:
Thanks for that change.

Here is a link to the code smell I was referring to, just as an FYI: https://github.com/troessner/reek/blob/master/docs/Boolean-Parameter.md

It wasn't clear to me in what sense the base64 encoding `Safe`?  As in safe from encoding issues?  I'm not necessarily against the name but I'd have to look it up if I just saw it to understand it.